File: app/src/main/java/org/kindone/willingtodo/data/TaskContextListItem.java
package org.kindone.willingtodo.data;

import org.kindone.willingtodo.recyclerlist.RecyclerListItem;

/**
 * Created by kindone on 2016. 12. 25..
 */

public class TaskContextListItem implements RecyclerListItem{
    private TaskContext context;

    public TaskContextListItem(TaskContext context) {
        this.context = context;
    }


    @Override
    public long getId() {
        return context.id;
    }

    @Override
    public String getTitle() {
        return context.name;
    }

    public TaskContext getTaskContext() {
        return context;
    }
}

File: app/src/main/java/org/kindone/willingtodo/data/TaskContextListItem.kt
package org.kindone.willingtodo.data

import org.kindone.willingtodo.recyclerlist.RecyclerListItem

/**
 * Created by kindone on 2016. 12. 25..
 */

class TaskContextListItem(val taskContext: TaskContext) : RecyclerListItem {

    override fun getId(): Long {
        return taskContext.id
    }

    override fun getTitle(): String {
        return taskContext.name
    }
}
